docs/tutorials/visibility_simulator.ipynb

```diff
@@ -57,12 +57,22 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        source_sim.uvdata.freq_array[0, 0] / 1e6,
Line #24.        source_sim.uvdata.freq_array[0, -1] / 1e6,
</pre><p>Update to <code>future_array_shapes</code> convention to fix the bug.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>woops, done</p>